packages/liveblocks-node/src/client.ts

```diff
@@ -48,6 +57,48 @@ type ThreadParticipants = {
   participantIds: string[];
 };
 
+export type RoomPermission =
+  | ["room:write"]
+  | ["room:read", "room:presence:write"];
```

Comment:
@nvie What I've done for now is create this `RoomPermission` type. I know `Session` also makes use of `Permission` type at some places and has the complete set of permissions. I haven't updated those for the sake of keeping this PR not too broad. But, I think it's worth taking a look at how our backend handles permission and updating these code accordingly.

Reply:
Really nice! I wanted to suggest this yesterday, but refrained from it because I didn’t want to be the nagging Puritan. Again, like I mentioned above, starting with the strictest types possible is a great starting position. Loosening is always possible for free, when it comes to input params.